Use eslintrc and prettierrc from shared and add husky config file

nicolai-nordic opened the pull request.

datenreisender reviewed (approved):
Looks good, thanks for the cleanup!

As you can see, I merged in master since it had progressed, so that GitHub does not complain about that anymore. I also added two small changes: Removed the stale entry in `.gitignore` and added a newline at end of `.huskyrc.json`.

nicolai-nordic commented:
> Looks good, thanks for the cleanup!
> 
> As you can see, I merged in master since it had progressed, so that GitHub does not complain about that anymore. I also added two small changes: Removed the stale entry in `.gitignore` and added a newline at end of `.huskyrc.json`.

Oh oops! I had changed that (i.e. stale .gitignore entry), but had to context switch before I commited, so I stashed and forgot to pop it. Thanks for fixing it!
